### PR TITLE
Bhavya

### PR DESCRIPTION
**Pull request by bordia98**
This Pull Request resolves Issue #<IssueNumber>

Description:
In this Pull Request we have done the following additions/edits for the code:

Note: Delete all the text including this!
* Make sure you replace `#<IssueNumber>` with the particular Issue that you are referencing with!
* Make sure you describe the Issue properly and ask as many people as possible to review your Pull Request so that it can be merged as soon as possible!

### CODE REVIEW

File: participants/bhavya.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Name - Bhavya
+year - 2nd year
+
```
Comment (MJ10) [inline participants/bhavya.txt]: Please make the previously requested changes

File: pythonprogram/infixtoprefix.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+s='A + B * C / F + D'
```
Comment (MJ10) [inline pythonprogram/infixtoprefix.py]: This does not belong to any issue. Please refer the Contributing Guidelines

**Comment by salman-bhai** [comment]
Are there any updates here or should we close the PR and label it as `spam`?

**Comment by salman-bhai** [comment]
spam!

**Pull request closed without merging**